src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -328,7 +328,14 @@ class HeaderHelpMenuUI extends Component<Props, State> {
         })}
         onClick={this.onMenuButtonClick}
       >
-        <EuiIcon type="help" size="m" />
+        <EuiIcon
+          type="help"
+          size="m"
+          title={intl.formatMessage({
+            id: 'core.ui.chrome.headerGlobalNav.helpMenuButtonTitle',
+            defaultMessage: 'Help menu',
```

Comment:
```suggestion
            defaultMessage: 'Help',
```

I think the agreed upon text is simply "Help" - @KrooshalUX can you confirm?

Reply:
It should be `Help`, yes.